src/test/resources/conf4

```diff
@@ -0,0 +1,23 @@
+misc:
```

Comment:
seems like this file is not taken into account for the test cases. could you also delete this file.